lib/msal-browser/src/request/AuthorizationCodeRequest.ts

```diff
@@ -10,4 +10,5 @@ export type AuthorizationCodeRequest = Partial<Omit<CommonAuthorizationCodeReque
     cloudGraphHostName?: string;
     msGraphHost?: string;
     cloudInstanceHostName?: string;
+    correlationId?: string;
```

Comment:
This is already included as part of `BaseAuthRequest` which `CommonAuthorizationCodeRequest` extends

Reply:
It is, but there it is required, this makes it optional.

Reply:
It's made optional by `Partial`

Reply:
Indeed! Missed that.